vcap_utils/vcap_utils/backends/base_openvino.py

```diff
@@ -196,7 +195,8 @@ def parse_detection_results(
             resize: Resize,
             label_map: Dict[int, str],
             min_confidence: float = 0.0,
-            boxes_output_name: str = None) -> List[DetectionNode]:
+            boxes_output_name: str = None,
+            frame_input_name: str = None) -> List[DetectionNode]:
```

Comment:
Should these be marked `Optional`, or is the annotation or default wrong?

Reply:
AFAIK it's conventional for keyword arguments to be assumed to be Optional[the-type] if you type hint it like this. 

Pycharm reads it that way, as does mypy [apparently](https://stackoverflow.com/questions/51055952/does-a-default-parameters-overwrite-type-hints-for-mypy)

I prefer it as well, since it's very clear what's happening on this line.

Reply:
Hmm I'd rather be explicit, (and use `--no-implicit-optional` if we were using mypy), but I'll leave this for now